doc/source/release-notes/1.6.1.rst

```diff
@@ -0,0 +1,39 @@
+.. _PBM-1.6.1:
+
+================================================================================
+*Percona Backup for MongoDB* 1.6.1
+================================================================================
+
+:Date: November 4, 2021
+:Installation: `Installing Percona Backup for MongoDB <https://www.percona.com/doc/percona-backup-mongodb/installation.html>`_
+
+Percona Backup for MongoDB is a distributed, low-impact solution for consistent backups of MongoDB
+sharded clusters and replica sets. This is a tool for creating consistent backups
+across a MongoDB sharded cluster (or a single replica set), and for restoring
+those backups to a specific point in time. 
+
+Release Highlights
+==================
+
+* Deprecated support for MongoDB 3.6. |PBM| remains compatible with MongoDB 3.6 and |PSMDB| 3.6; however, further enhancements and bug fixes are no longer tested against this version. 
+* Improved backup and PITR routines alignment by using sequential ``delete-pitr``/``install-backup`` operations instead of in-memory backup intent. This fixes the inability of a backup to start.
+* Added support for automated access to S3 buckets using an EC2 instance profile. When |PBM| is deployed using an EC2 instance, EC2 environment variables and metadata are used for S3 authentication saving you from explicitly specifying S3 credentials in the |PBM| configuration file. This comes handy for architectures deployed using the services like Amazon EC2, kiam, kube2iam or irsa.  
+* Extended logging for ``pbm-agents``. This improves user experience with PBM.
+
+Improvements
+================================================================================
+
+* :jirabug:`PBM-714`: Improve backup and PITR routines alignment algorithm.
```

Comment:
I have changed the issue type to "bug". It is not an improvement.
Lets rephrase it as well to "Fix backup and PITR routines alignment algorithm to avoid backup failures" ?